flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -1578,9 +1578,11 @@ bool OmpAttributeVisitor::Pre(
     const parser::OpenMPSimpleStandaloneConstruct &x) {
   const auto &standaloneDir{
       std::get<parser::OmpSimpleStandaloneDirective>(x.t)};
+  const auto &parentContext{GetContextIf()};
```

Comment:
Nit: Since it's not used anywhere else, why not move this definition to the `if` body where it will be used?

Reply:
I saved the parentContext in a variable before the `scan`'s context was pushed @line 1591.